callback_var_test.go

```diff
@@ -0,0 +1,64 @@
+package goflags
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCallbackVarPositive(t *testing.T) {
```

Comment:
We can change `Positive/Negative` to more meaningful names such as `SuccessfulCallback`. If we opt for this change, even if unrelated to this PR, can we change the same naming style for the Enum type?

Reply:
I think we can change for Enum type as well since that test case is not re-used in anywhere.